### ISSUE TITLE
A consultation on Therapy-Induced Evolution of Human Lung Cancer Revealed by Single-Cell RNA Sequencing (2020, Cell 182, 1232–1251)

### ISSUE THREAD

**Issue by JovannyChen** [czbiohub/scell_lung_adenocarcinoma#4]
Dear Prof.Maynard,

We are reading your recent significant work Therapy-Induced Evolution of Human Lung Cancer Revealed by Single-Cell RNA Sequencing (2020, Cell 182, 1232–1251).  And we admire this interesting article. 

For the purpose of discussion, we have a question: When we perform quality control for the scRNA-seq data, what proportion did you use to control the mitochondrial gene for the samples. As previouly described in the other studies, mito genes < 10% or 20%. However, as we all know, the sample from patients after targeted drugs might experience More necrosis, which possibly leads to more mito genes. Thus, how should we control this condition to acquire better cell number to reflect actual tumor or tumor microenvironment. We could not find the mito genes control in your codes  https://github.com/czbiohub/scell_lung_adenocarcinoma
 
Thank you very much.

Sincerely,

**Comment by AMaynard10** [comment]
Hello JovannyChen. Thank you for your interest in the data. The reference we used for alignment did not include mito genes, so in our case we did not do our QC based on mito% (and therefore side step the problem you noted above). Instead we did QC based on number of unique genes and reads, for example in script 2.1  we filter cells so that remaining cells have nGenes >= 500 and nReads >= 50000. Happy to discuss more.
